src/Bundle/DependencyInjection/SyliusResourceExtension.php

```diff
@@ -73,6 +74,9 @@ public function prepend(ContainerBuilder $container): void
     {
         $config = ['body_listener' => ['enabled' => true]];
         $container->prependExtensionConfig('fos_rest', $config);
+
+        $config = ['orm' => ['default_repository_class' => EntityRepository::class]];
```

Comment:
Just a note for other reviewers. This line will by default set our `Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository` as the default repository class as stated in the description. But it means that all non-resource-based doctrine entities will have two additional functions in their default repository class (add and remove), which could not be used, due to ResourceInterface typehint

Reply:
i made a version without the need to set this config. please check:
https://github.com/ecoco/SyliusResourceBundle/compare/%23217_duplicate_initialisation_of_repositories...ecoco:%23217_duplicate_initialisation_of_repositories_v2

and let me know what you think. If you like it should i just update this pull request or create a new one?